tools/git-change-exec/actions-test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// Copyright (c) 2024 Zededa, Inc.
```

Comment:
No need for two SPDX headers =)

Reply:
ups, that happened when I tested it on itself ...